docs/advanced.md

```diff
@@ -399,6 +399,22 @@ client = httpx.Client(timeout=timeout)
 response = client.get('http://example.com/')
 ```
 
+## Pool limit configuration
+
+You can control the connection pool size using the `pool_limits` keyword
+argument on the client. It takes instances of `httpx.PoolLimits` which define:
+
+- `soft_limit`, number of allowable keep-alive connections, or `None` to always
+allow. (Defaults 10)
+- `hard_limit`, maximum number of allowable connections, or` None` for no limits.
+(Default 100)
```

Comment:
This highlights to me that we're using the clearer "max_keepalive" and "max_connections" in `httpcore`.

I wonder if we should deprecate across to that naming for 1.0?

Reply:
Yeah, I agree it's clearer.